### DISCUSSION TITLE
tree-sitter-powershell

### DISCUSSION THREAD

**Discussion by citronneur** [tree-sitter/tree-sitter#2688] [Grammars]
Hello tree-sitter team,

At Airbus CERT we are glad to release a tree-sitter grammar for Powershell : https://github.com/airbus-cert/tree-sitter-powershell

This grammar cover the entire language until version 7.3 (the last one). Is it possible to add a ref to https://tree-sitter.github.io/ ?

Thanks in advance !

**Comment by sogaiu** [comment]
I think the usual pattern is to submit a PR like [this](https://github.com/tree-sitter/tree-sitter/pull/2459).

Apparently, these types of PRs are [merged in batches from time to time](https://github.com/tree-sitter/tree-sitter/pull/2459#issuecomment-1694435031).

On a side note, I guess the development of the grammar [here](https://github.com/PowerShell/tree-sitter-PowerShell) hasn't been active recently...There appear to be a number of forks, some of which have more commits, but may be those are not as complete either?

**Reply by citronneur** [reply]
Thanks!
